site/content/en/docs/Tasks/namespaces_and_names.md

```diff
@@ -1,8 +1,119 @@
 ---
 title: "Namespaces and Names"
 linkTitle: "Namespaces and Names"
-weight: 1
-date: 2017-01-05
+weight: 3
+date: 2023-10-14
 description: >
   Working with Namespaces and Names
----
\ No newline at end of file
+---
+
+The Namespace can be set for all Resources in a project by adding the `namespace` entry to the `kustomization.yaml` file. Consistent naming conventions can be applied to Resource Names in a project with the `namePrefix` and `nameSuffix` fields.
+
```

Comment:
I removed this content.

Reply:
Instead of leaving them completely blank, I think it would be nice to reference the corresponding Task section,

https://github.com/kubernetes-sigs/kustomize/pull/5383/commits/20e05bcadb5e489fc3471513c280965b9a1bb05f

Reply:
I added a link in to the Tasks. The Reference will be updated separately as part of https://github.com/kubernetes-sigs/kustomize/issues/5352.